uber/automated_emails.py

```diff
@@ -47,9 +47,12 @@ def render(self, x):
         model = getattr(x, 'email_model_name', x.__class__.__name__.lower())
         return render('emails/' + self.template, dict({model: x}, **self.extra_data))
 
+    def is_html(self):
+        return not self.template.endswith('.txt')
+
     def send(self, x, raise_errors=True):
         try:
-            format = 'text' if self.template.endswith('.txt') else 'html'
+            format = 'text' if self.is_html() else 'html'
```

Comment:
I think you've switched around the logic here. Shouldn't this be `if not self.is_html()`?